tracks/csharp/exercises/nucleotide-count/mentoring.md

```diff
@@ -0,0 +1,57 @@
+### Reasonable solutions
+
+### Using a foreach loop
```

Comment:
Increase to level 4:
```suggestion
#### Using a foreach loop
```

Reply:
Resolved in 3bf9ce67cbd9ef4eef08e3718ec54ef4e11ed10c